jenkins/Test.groovy.j2

```diff
@@ -266,8 +266,13 @@ stage('Test') {
             )
           }
           {{ m.upload_artifacts(tag='docs', filenames=["docs.tgz"]) }}
+          def docs_path = "tvm-jenkins-artifacts-prod/tvm/${env.BRANCH_NAME}/docs/"
```

Comment:
Can this use `s3_prefix` which also incorporates `BUILD_NUMBER`?

See:
https://github.com/apache/tvm/blob/70884e957aa5c8de9c02c25a14d30563d7300cb9/Jenkinsfile#L97-L98

Reply:
Then I will need the jenkins build number in github actions, which I am not sure how to obtain.

Reply:
Hmm, maybe @driazati can advise on how tricky that is? 😸